src/app/[locale]/seller/registration/page.tsx

```diff
@@ -227,58 +288,7 @@ const SellerRegistrationForm = () => {
             }}
           />
         </Link>
-        <div className="mb-4 mt-3 ml-auto w-min">
```

Comment:
Not sure why the collapsible elements were removed.  Please revert and restore them.  Thanks!

Reply:
Yeah that will be reversed